{"name": "Проверка получения ошибки аутентификации при вводе невалидных данных", "status": "passed", "description": "В тест по очереди передаются наборы данных с несуществующим логином или неверным паролем. Проверяются код и тело ответа.", "steps": [{"name": "Отправка POST запроса на авторизацию с невалидными данными", "status": "passed", "start": 1760865213137, "stop": 1760865213604}], "parameters": [{"name": "nonexistent_credentials", "value": "{'login': 'Least.668', 'password': '0*)#$5EtFf'}"}], "start": 1760865213136, "stop": 1760865213605, "uuid": "2510620f-ee8d-4383-9009-8e024f6b801e", "historyId": "b2cb17860092fb521392dc760d715d20", "testCaseId": "c51f304cd3dfff9c90b10c807d062f2c", "fullName": "tests.tests_couriers.test_courier_login.TestCourierLogin#test_courier_login_nonexistent_data_not_found", "labels": [{"name": "parentSuite", "value": "tests.tests_couriers"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "Sergei"}, {"name": "thread", "value": "5332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_couriers.test_courier_login"}], "titlePath": ["tests", "tests_couriers", "test_courier_login.py", "TestCourierLogin"]}